doc/rtd/reference/datasources/vmware.rst

```diff
@@ -148,6 +148,30 @@ Features
 
 This section reviews several features available in this datasource.
 
+Graceful rpctool fallback
+-------------------------
+
+The datasource initially attempts to use the program ``vmware-rpctool`` if it
+is available. However, if the program returns a non-zero exit code, then the
+datasource falls back to using the program ``vmtoolsd`` with the ``--cmd``
+argument.
+
+On some older versions of ESXi and open-vm-tools, the ``vmware-rpctool``
+program is much more performant than ``vmtoolsd``. While this gap was
+closed, it is not reasonable to expect the guest where Cloud-Init is running to
```

Comment:
Same as above
```suggestion
closed, it is not reasonable to expect the guest where cloud-init is running to
```

Reply:
Is that something that can/should be lintable?

Reply:
Good question! We'd need to extend a linter with our own rules which we haven't done before. Have you used anything that makes it easy?

Reply:
For markdown, but not for rst. Let me dig into what you're using for rst.